.github/workflows/docker-openmetadata-server.yml

```diff
@@ -11,6 +11,7 @@
 
 name: docker-openmetadata-server docker
 on:
+  workflow_dispatch:
```

Comment:
We are missing an input `tag` here! This input will be used for specifying docker image tag when running on demand!